src/MicrosoftTeams.ts

```diff
@@ -421,22 +422,61 @@ namespace microsoftTeams
             let link = document.createElement("a");
             link.href = authenticateParameters.url;
 
-            let messageId = sendMessage(
-                "authentication.authenticate",
-                link.href,
-                authenticateParameters.width,
-                authenticateParameters.height);
-            callbacks[messageId] = (success: boolean, response: string) =>
+            // Clear any previous success/failure values
+            localStorage.removeItem("authentication.success");
+            localStorage.removeItem("authentication.failure");
+
```

Comment:
This is now moot since we're not using localStorage

---
In reply to: [87050549](https://github.com/OfficeDev/microsoft-teams-library-js/pull/10#discussion_r87050549) [](ancestors = 87050549)